stdlib/public/Python/Python.swift

```diff
@@ -1253,6 +1253,24 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : Sequence {
+  public struct Iterator : IteratorProtocol {
+    let pyObjIterator: PythonObject
```

Comment:
This property should be `private` or `fileprivate` because it is only used within this extension.